UKHomeOffice/sgmr-service
# SPCR-418 Update content for Voyage Plan Confirmation Page

### AC / Description of changes
To add guidance text to the confirmation page to inform the users to check their emails for the reference to the voyage report submission.

Given:  The user has submitted the Voyage report.
When:  The user is presented with the Confirmation Page ‘..your VP has been sent successfully…’  
Then:  That confirmation page ALSO includes the following guidance to the user.

Suggested wording - Please check your registered email account for the submission reference number associated with this voyage report and quote this on any correspondence with Border Force

### To Test
- Create a voyage plan
- Submit the voyage plan
- > The content should be updated with the wording above, under the "What happens next" section

### Notes

---

**Remember to merge to feature branches not `master`.**
